common/src/test/resources/json/guild/guildmember.json

```diff
@@ -3,5 +3,6 @@
   "roles": [],
   "joined_at": "2015-04-26T06:26:56.936000+00:00",
   "deaf": false,
-  "mute": false
+  "mute": false,
+  "flags": 0
```

Comment:
I think it would be better to use non-zero values

Reply:
zero values are used in the other tests, hence i used zero here